docs/fundamentals/code-analysis/quality-rules/ca1062.md

```diff
@@ -48,9 +48,18 @@ To fix a violation of this rule, validate each reference argument against `null`
 
 You can suppress a warning from this rule if you are sure that the dereferenced parameter has been validated by another method call in the function.
 
-## Configurability
+## Configure code to analyze
 
-This rule has the following configurable options.
+This rule has the following configurable options that can be configured for just this rule, for all rules, or for all rules in this category (Design). For more information, see [Code quality rule configuration options](../code-quality-rule-options.md).
+
+Use the following options to configure which parts of your codebase to run this rule on.
+
+- [Exclude specific symbols](#exclude-specific-symbols)
+- [Exclude specific types and their derived types](#exclude-specific-types-and-their-derived-types)
```

Comment:
I think we should probably also add:

```suggestion
- [Null check validation methods](#null-check-validation-methods)
```

Otherwise, make "Null check validation methods" an H2.